Description: Upstream changes introduced in version r71-1nano
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 nand-bin-check (r71-1nano) unstable; urgency=low
 .
   * Initial release
 .
 The person named in the Author field signed this changelog entry.
Author: Christopher Roy Bratusek <zanghar@freenet.de>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- nand-bin-check-r71.orig/main.cpp
+++ nand-bin-check-r71/main.cpp
@@ -1,12 +1,12 @@
-#include "../WiiQt/includes.h"
-#include "../WiiQt/nandbin.h"
-#include "../WiiQt/sharedcontentmap.h"
-#include "../WiiQt/uidmap.h"
-#include "../WiiQt/tools.h"
-#include "../WiiQt/tiktmd.h"
-#include "../WiiQt/settingtxtdialog.h"
-#include "../WiiQt/u8.h"
-#include "../WiiQt/keysbin.h"
+#include "WiiQt/includes.h"
+#include "WiiQt/nandbin.h"
+#include "WiiQt/sharedcontentmap.h"
+#include "WiiQt/uidmap.h"
+#include "WiiQt/tools.h"
+#include "WiiQt/tiktmd.h"
+#include "WiiQt/settingtxtdialog.h"
+#include "WiiQt/u8.h"
+#include "WiiQt/keysbin.h"
 
 //yippie for global variables
 QStringList args;
